### ISSUE TITLE
Uncaught TypeError: Cannot read property 'style' of undefined

### ISSUE THREAD

**Issue by fobos** [olmokramer/atom-minimap-codeglance#18]
1. Just open an empty Atom
2. Place mouse pointer above empty minimap pane

**Atom Version**: 1.1.0
**System**: Mac OS X 10.11.1
**Thrown From**: [minimap-codeglance](https://github.com/olmokramer/atom-minimap-codeglance) package, v0.4.4
### Stack Trace

Uncaught TypeError: Cannot read property 'style' of undefined

```
At /Users/illbullet/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68

TypeError: Cannot read property 'style' of undefined
    at minimap-codeglance.showLineNumbers (/Users/illbullet/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68:20)
    at /Users/illbullet/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:13:22
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:549:7)
    at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:129:21)
    at Object.initialize (/Users/illbullet/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:12:19)
    at minimapCodeglance (/Users/illbullet/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:154:35)
    at Object.activatePlugin (/Users/illbullet/.atom/packages/minimap-codeglance/lib/main.js:31:32)
    at Main.module.exports.PluginManagement.updatesPluginActivationState (/Users/illbullet/.atom/packages/minimap/lib/mixins/plugin-management.coffee:92:16)
    at /Users/illbullet/.atom/packages/minimap/lib/mixins/plugin-management.coffee:112:24
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:549:7)
    at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:129:21)
    at Main.module.exports.PluginManagement.registerPluginControls (/Users/illbullet/.atom/packages/minimap/lib/mixins/plugin-management.coffee:110:55)
    at Main.module.exports.PluginManagement.registerPlugin (/Users/illbullet/.atom/packages/minimap/lib/mixins/plugin-management.coffee:38:14)
    at Object.consumeMinimapServiceV1 (/Users/illbullet/.atom/packages/minimap-codeglance/lib/main.js:20:18)
    at Provider.module.exports.Provider.provide (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/provider.js:30:52)
    at /Applications/Atom.app/Contents/Resources/app.asar/node_modules/service-hub/lib/service-hub.js:55:26
```
### Commands

```
```
### Config

``` json
{
  "core": {}
}
```
### Installed Packages

``` coffee
# User
editorconfig, v1.2.2
linter, v1.10.0
linter-eslint, v4.0.0
minimap, v4.16.0
minimap-bookmarks, v0.1.0
minimap-codeglance, v0.4.4
minimap-find-and-replace, v4.3.2
minimap-git-diff, v4.1.8
project-manager, v2.6.5

# Dev
No dev packages
```

**Comment by chrissphinx** [comment]
Also occurring under Yosemite, definitely related to #17.
1. Start Atom with minimap and minimap-codeglance enabled
2. Error is thrown:

**Atom Version**: 1.1.0
**System**: Mac OS X 10.10.5
**Thrown From**: [minimap-codeglance](https://github.com/olmokramer/atom-minimap-codeglance) package, v0.4.4

### Stack Trace

Uncaught TypeError: Cannot read property 'style' of undefined

```
At /Users/sphinx/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68

TypeError: Cannot read property 'style' of undefined
    at minimap-codeglance.showLineNumbers (/Users/sphinx/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68:20)
    at /Users/sphinx/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:13:22
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:549:7)
    at Config.module.exports.Config.observe (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:129:21)
    at Object.initialize (/Users/sphinx/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:12:19)
    at minimapCodeglance (/Users/sphinx/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:154:35)
    at Object.activatePlugin (/Users/sphinx/.atom/packages/minimap-codeglance/lib/main.js:31:32)
    at Main.module.exports.PluginManagement.updatesPluginActivationState (/Users/sphinx/.atom/packages/minimap/lib/mixins/plugin-management.coffee:92:16)
    at /Users/sphinx/.atom/packages/minimap/lib/mixins/plugin-management.coffee:112:24
    at Config.module.exports.Config.observeKeyPath (/Applications/Atom.app/Contents/Resources/app.asar/src/config.js:549:7)
```

### Commands

```
```

### Config

``` json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ],
    "disabledPackages": [
      "vim-mode",
      "language-latex",
      "atom-beautify"
    ]
  }
}
```

### Installed Packages

``` coffee
# User
minimap, v4.16.0
minimap-codeglance, v0.4.4

# Dev
No dev packages
```

**Comment by anderson-arlen** [comment]
Also occurring under Linux Mint 17.2
1. ...Start Atom with minimap and minimap-codeglance enabled
2. ...Error is thrown...

**Atom Version**: 1.1.0
**System**: Linux Mint
**Thrown From**: [minimap-codeglance](https://github.com/olmokramer/atom-minimap-codeglance) package, v0.4.4

### Stack Trace

Uncaught TypeError: Cannot read property 'style' of undefined

```
At /home/arlen/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68

TypeError: Cannot read property 'style' of undefined
    at minimap-codeglance.showLineNumbers (/home/arlen/.atom/packages/minimap-codeglance/lib/minimap-codeglance-element.js:68:20)
    at /home/arlen/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:13:22
    at Config.module.exports.Config.observeKeyPath (/usr/share/atom/resources/app.asar/src/config.js:549:7)
    at Config.module.exports.Config.observe (/usr/share/atom/resources/app.asar/src/config.js:129:21)
    at Object.initialize (/home/arlen/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:12:19)
    at minimapCodeglance (/home/arlen/.atom/packages/minimap-codeglance/lib/minimap-codeglance.js:154:35)
    at Object.activatePlugin (/home/arlen/.atom/packages/minimap-codeglance/lib/main.js:31:32)
    at Main.module.exports.PluginManagement.updatesPluginActivationState (/home/arlen/.atom/packages/minimap/lib/mixins/plugin-management.coffee:92:16)
    at /home/arlen/.atom/packages/minimap/lib/mixins/plugin-management.coffee:112:24
    at Config.module.exports.Config.observeKeyPath (/usr/share/atom/resources/app.asar/src/config.js:549:7)
```

### Commands

```
```

### Config

``` json
{
  "core": {
    "themes": [
      "atom-material-ui",
      "one-dark-syntax"
    ],
    "autoHideMenuBar": true
  }
}
```

### Installed Packages

``` coffee
# User
atom-material-ui, v0.8.0
atomatigit, v1.5.5
auto-fold, v0.2.0
autohide-tree-view, v0.25.1
clipboard-plus, v0.5.1
color-picker, v2.0.13
file-icons, v1.6.11
highlight-line, v0.11.0
highlight-selected, v0.10.1
linter-jade, v0.3.2
minimap, v4.16.0
minimap-autohide, v0.10.1
minimap-bookmarks, v0.1.0
minimap-codeglance, v0.4.4
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.1
minimap-pigments, v0.1.7
pigments, v0.18.1
regex-railroad-diagram, v0.10.4
tree-view-git-status, v0.2.2

# Dev
No dev packages
```

**Comment by olmokramer** [comment]
Thanks for the reports everyone. This may be related to #17 as @chrissphinx mentions, but just in case I've added an extra check where this specific error happens. I'll release the patch after I've done some more testing.

**Comment by davisonio** [comment]
Is anyone still getting this issue?

**Comment by sbrl** [comment]
@davisonio I haven't seen this in a while, but I haven't been looking out for it.

**Comment by olmokramer** [comment]
Hi everyone, I'm sorry for the extremely late reply. As you have probably noticed I've stopped maintaining my Atom plugins a few years ago. I got very frustrated with how Atom was maintained and started ignoring all of my plugins. I'm sorry for that. I won't install Atom to test and mergeer PRs, but I'd be happy to transfer my repositories to anyone that wants to maintain any of my plugins. Again, I'm sorry.

**Comment by sbrl** [comment]
I see. I don't think I'm in a position to take on maintenance of this atom plugin at this time.

If you don't mind me asking, what issue did you have with Atom, and which editor did you switch to?

**Comment by olmokramer** [comment]
@sbrl The main issue I had with Atom was that it constantly broke my (and lots of others) workflow, and then refusing to add an option or API to get the old behaviour back. So then I'd have to hack around it using internal API's to get the old behaviour back, but they were also constantly changing those constantly. In the end I got so frustrated with it that I dropped Atom entirely. Now I'm using Vim.
